_episodes/09-pull-requests.md

```diff
@@ -1,51 +1,81 @@
+Instead of making changes directly to a repository you can suggest changes to a repo.
+This can be useful if you don't have permission to modify a repository directly or
 you want someone else to review your changes.
 
 For this lesson we will be working on the `countries` repository together.
-Open the github link for the `countries` repo
-at [https://github.com/McMahonLab/countries](https://github.com/McMahonLab/countries).
+Open the github link for the `countries` repo provided by the instructor 
+in your browser window.
 
 ![](../fig/github_screenshot_upstream_repo.png)
 
-Then click the fork button in the upper right hand conner.  
+> ## Repo owner differences
+>
+> You may have noticed that the `countries` repo in this lesson's pictures 
+> is owned by the 'McMahonLab' organization and this doesn't match
+> the address you were given.
+> This is to be expected because this will differ depending on 
+> what organization your instructor used to setup the `countries` repo.
+> 
+> You will also see your username where the 'sstevens2' is in the picutres.
+> 
+{: .callout}
 
-Next we are going to setup the connections to both our forked version and the authoritative version we forked it from.
+Once at the `countries` repo, click the **Fork** button which can be found
+in the upper right hand conner of the window. 
+Forking the repository makes us each our own copy of the repo in our github 
+account which we can edit.
 
 ![](../fig/github_screenshot_upstream_forking.png)
 
-First we will clone the repo from our forked version so that our version is setup as the 'origin' connection.
-Copy the web address for your forked repo (from the web address line or click 'clone and download' and copy that).
+Next we need to get this repo on our local computer and
+setup connections from our computer to both our forked version 
+and the authoritative version we forked it from.
+
+First we will **clone** the repo from our forked version.
+The clone command does two things:
+	1. Copies the repo to your local computer
+	2. Sets up a remote called 'origin' between your computer and the github repo
+Copy the web address for your forked version of repo 
+(from the web address line or click 'clone and download' and copy that).
 
 ![](../fig/github_screenshot_cloneOrigin.png)
 
-Open your terminal and navigate to a folder you'd like to hold this repo, maybe your `Desktop`.
-Once there you can clone the repo using the link we just copied.
+In terminal or Gitbash, navigate to a folder you'd like to hold this repo,
+we will place it on our `Desktop`.
+Once there you can use the `clone` command with the link you copied as the first argument.
 
 ~~~
-$ cd Desktop
+$ cd ~/Desktop
 $ git clone https://github.com/USERNAME/countries.git
 ~~~
 {: .bash}
 
+> ## Why does the command above say 'USERNAME'?
+>
+> So that we can't copy the command above and accidently clone someone else's
+> version of countries to our computer the command above uses the placeholder
```

Comment:
Some punctuation is needed on this line between "computer" and "the". Either a period or comma will do I think.